script.js

```diff
@@ -0,0 +1,25 @@
+var block = document.getElementById("test");
+var left = 0;
+var top1 = 0;
+var bottom = 0;
+var right = 0;
+
+document.onkeydown = function (event){
+  console.log(event);
+  if (event.key == "ArrowRight"){
+    block.style.left = left + "px";
+    left++; 
+  }
+  if (event.key == "ArrowDown"){
+    block.style.top = top1 + "px";
+    top1++; 
+  }
+  if (event.key == "ArrowUp"){
```

Comment:
wrong logic. 
to move a car to the top or to the left you should do `block.style.top--` or  `block.style.left--`
because there is no Right and Bottom :(